tools/make/build-container.mk

```diff
@@ -34,7 +34,7 @@
 # variable names should be passed through to the container.
 
 USE_CONTAINER       ?= 0
-BUILD_IMAGE_VERSION ?= v0.1.2
+BUILD_IMAGE_VERSION ?= v0.1.3
```

Comment:
Normally we update this in a follow up PR after the image is already available. Should this be pulled out of this PR?

Reply:
Rolled back